### ISSUE TITLE
Convert `textile` to `asciidoc`

### ISSUE THREAD

**Issue by bartoszmajsak** [arquillian/arquillian.github.com#322]
Would be great to leverage `asciidoc` with all its great toolchain to render our posts, guides etc. Maybe `pandoc` can automate it?

**Comment by aslakknutsen** [comment]
@mojavelinux Had many of the Guides converted.. ?

**Comment by mojavelinux** [comment]
I had converted a few of the guides as a prototype, which I pushed as gists:

* https://gist.github.com/mojavelinux/4654394
* https://gist.github.com/mojavelinux/4654385

(some of the content may be out of date by now).

@bartoszmajsak  I'm assuming you mean pandoc for the conversion to AsciiDoc, right? If I remember correctly, @aslakknutsen, you had hacked together a Textile to AsciiDoc converter. I had also created one, but it's pretty hackish: https://gist.github.com/mojavelinux/4708592

**Comment by bartoszmajsak** [comment]
Thanks buddy! I will investigate.

**Comment by mojavelinux** [comment]
One of the main reasons I had put off switching to AsciiDoc because, at the time, I was still waiting for extensions and custom templates to be merged into Asciidoctor. Now that they are in and mature, you should have all the flexibility you need to create the pages using the exact same HTML, which the current CSS & JavaScript expects. Taking this route could save you from having to mess with that stuff. (In effect, just an input change, not an output change). You also have the option to create custom macros to handle some of the boilerplate content (such as the release info table). (See https://github.com/asciidoctor/asciidoctor-extensions-lab for a bunch of ideas).

**Comment by aslakknutsen** [comment]
@mojavelinux Yes, forgot about that. I made a RedCloth Formatter that can format Textile to Asciidoc: https://github.com/aslakknutsen/redcloth/tree/asciidoc

Don't remember the state, but I think this was one of the test outputs:
https://gist.github.com/aslakknutsen/5d159a6e761c114d0460
